README.md

```diff
@@ -15,18 +15,19 @@ uutils coreutils
 -----------------------------------------------
 
 uutils is an attempt at writing universal (as in cross-platform) CLI
-utils in [Rust](http://www.rust-lang.org). This repo is to aggregate the GNU
-coreutils rewrites.
+utilities in [Rust](http://www.rust-lang.org). This repository is intended for
+aggregating the GNU coreutils rewrites.
```

Comment:
This sounds kinda weird to me.  I’d prefer `intended to aggregate GNU coreutils rewrites` or something like that.